packages/internal-test-helpers/lib/maybe-expect-deprecation.ts

```diff
@@ -0,0 +1,16 @@
+import { ExpectDeprecationFunc } from './ember-dev/deprecation';
+import { Message } from './ember-dev/utils';
+
+declare let expectDeprecation: ExpectDeprecationFunc;
+
+export default function maybeExpectDeprecation(
+  featureFlag: boolean | null,
+  callback: () => void,
+  message: Message
+): void | undefined | Message {
```

Comment:
```suggestion
): void {
```

This should be returning `void` only, since none of the branches actually returned anything.